lib/services/presets_test.go

```diff
@@ -688,6 +688,7 @@ func TestAddRoleDefaults(t *testing.T) {
 									types.KindClusterNetworkingConfig,
 									types.KindDatabase,
 									types.KindDevice,
+									types.KindDiscoveryConfig,
```

Comment:
I ran into this same test failure when I tried to fix this test for another resource lol
You just need to move this into the "expected" resource and put it in a separate rule as below, e.g.,
```
types.NewRule(types.KindDiscoveryConfig, RW()),
```